src/main/NWN/API/Scripts/CallInfo.cs

```diff
@@ -26,10 +29,16 @@ public ScriptParams ScriptParams
     /// </summary>
     public string ScriptName { get; }
 
+    /// <summary>
+    /// Gets the type of event that called the script that is being executing.
```

Comment:
```suggestion
    /// Gets the event that triggered the execution of this script.
```

Minor nit about the docs, otherwise LGTM.